src/go/k8s/tests/e2e/create-topic-given-issuer-with-client-auth/00-assert.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: cert-manager.io/v1
```

Comment:
in general, it is easier for the reviewer to either:

1. functionality + test in the same commit or 
2. functionality and test in back-to-back commits to not loose context and flow

Bunching tests at the end makes one glance at the test only and not look at them in detail.

Reply:
well I get that with unit tests but I feel like these e2e tests in yaml are better reviewed separately like this. But I can ask other guys what they prefer :)